README.md

```diff
@@ -1,64 +1,123 @@
 
-OpenTargets also holds periodic releases, which happen approximately every two months. Data for every release comes from several data providers. There are several requirements for submitting data to OpenTargets:
-* It must be represented in the form of “evidence strings”. These are JSON strings describing:
-  + Genes the variant affects
-  + Functional consequence of the variant on the gene
-  + Traits (usually diseases) associated with the variant, such as “parkinson disease” or “age-related macular degeneration”.
-  + Other information about the variant and source, such as related publications
-* The variant data must be synchronised with a specific version of external data sources, for example Ensembl.
+### Annotation pipeline
 
-## Role of EVA
-ClinVar data is highly valuable, but in its original form is not suitable for submission to OpenTargets. EVA is registered as one of the submitters for OpenTargets. For every OpenTargets release, the EVA processes ClinVar records (variants), curates the result and submits it to OpenTargets in the form of evidence strings. This allows for the up-to-date ClinVar data to be integrated into the OpenTargets platform.
+This will annotate variants with genes and functional consequences, and annotate traits with ontology terms using an existing mappings file.
+It outputs the results as an annotated XML file.
 
-Approximately one month before the submission deadline, OpenTargets will contact their submitters and specify the requirements for the next release. At this point the EVA can start executing the main submission protocol (see below). Once the data is ready, it is submitted to OpenTargets, and then the same will happen with the next release. Most of the actions in the pipeline are automated.
+```bash
+# Directory to run annotation pipeline
+export ANNOTATION_ROOT=
 
+# Create directories for data processing
+mkdir -p ${ANNOTATION_ROOT}
+cd ${ANNOTATION_ROOT}
+mkdir -p gene_mapping logs
 
+# Run the nextflow pipeline, resuming execution of previous attempt if possible.
+nextflow run ${CODE_ROOT}/cmat/output_generation/pipeline.nf \
+  --output_dir ${ANNOTATION_ROOT} \
+  --mappings ${LATEST_MAPPINGS} \
+  -resume
+```
 
-# Workflow diagram
+By default, the pipeline will download and annotate the latest ClinVar XML dump from [FTP](https://ftp.ncbi.nlm.nih.gov/pub/clinvar/xml/). If you want to run it on an existing XML file, you can pass it via the `--clinvar` flag.
 
-![](docs/workflow-diagram/workflow.png)
+### Trait curation
 
-See details [here](docs/workflow-diagram) about how to regenerate this diagram from source.
+These are processes to update the trait mappings used by the annotation pipeline and should be performed regularly to ensure new ClinVar data is mapped appropriately.
 
-There is also a [presentation](https://docs.google.com/presentation/d/1kr1orv08ZGnPGKNu6vQk4wFYIrufu_iIDf-drCc21vY) describing the workflow in more detail.
+#### Automatic mappings and curation spreadsheet generation
```

Comment:
These steps are also described in the [manual curation](https://github.com/EBIvariation/CMAT/tree/master/docs/manual-curation) part of the docs. 
Do you want to have them in two places ?

Reply:
That's a good point, probably not... I think the main difference is the environment variable setup is a bit simpler in this version and it omits the feedback steps which are somewhat EVA-specific. But maybe it's better to just link to those docs and mention here that those steps are optional.

Reply:
So I've left these instructions to try to reduce friction for first-time users (with perhaps some minimal maintenance cost on us) while generally trying to distinguish Open Targets SOPs from CMAT documentation... I think it still needs some work but hopefully it's better at least.